app/templates/components/events/view/overview/event-sponsors.hbs

```diff
@@ -6,20 +6,15 @@
   <div class="header">{{t 'Event sponsors'}}</div>
 </div>
 <div class="content">
-  {{events/events-table
-    columns=columns
-    data=data
-    store=store
-    query=query
-    isNotStoreQuery = true
-    modelName = modelName
-    useNumericPagination=true
-    showGlobalFilter=true
-    showPageSize=true
-    moveToDetails='moveToDetails'
-    editEvent='editEvent'
-    deleteSponsor=deleteSponsor
-    editSponsor=editSponsor
-    customGlobalFilter='name'
-  }}
+  {{tables/default columns=columns
```

Comment:
that's because you have named the it sponsorsColumns in definition.

Reply:
@CosmicCoder96 I noticed that earlier, But when the `Model Tables` were used name was same, Moreover when I changed the `columns=sponsorsColumns` the column stopped rendering too ?

Reply:
@kushthedude Stop comparing anything with model tables, it's not relevant if we are not using them.
Also @kushthedude you are making changes in the controller of one route, and editing template of another, it doesn't work like that.

Reply:
@CosmicCoder96 Where should I define the `Ember Table`, In the `component/event/.../event-sponsors.js`?
Also isn't the route for `event-sponsors` is same as `event/view/index`?